libs/application/templates/financial-aid/src/fields/Status/ApplicantStatus.tsx

```diff
@@ -57,12 +59,12 @@ const ApplicantStatus = ({ application }: FAFieldBaseProps) => {
         created={
           currentApplication?.created
             ? new Date(currentApplication.created)
-            : application.created
+            : new Date(application.created)
         }
         modified={
           currentApplication?.modified
             ? new Date(currentApplication.modified)
-            : application.modified
+            : new Date(application.modified)
```

Comment:
Since we need to change both to new Date, maybe Aidamount in props created and modified should take string, and change it there, so we dont have to change everyone and remember to change it, we have above changed it four times 🙃